docs/user-guide/cli-db2plugin.md

```diff
@@ -187,47 +200,47 @@ Alternatively, you can create a db2 profile manually by adding a section that co
     }
     ```
 
-4. Save the file
+4. Save the file.
 
-You can now use your profile when you issue commands in the db2 command group.
+    You can now use your profile when you issue commands in the db2 command group.
```

Comment:
Db2

Reply:
In this instance, would it be clearer if we changed this to something like "You can now use your profile when you issue `zowe db2` commands"? 

"the db2 command group" refers to Zowe CLI commands beginning with `zowe db2` like `zowe db2 export table` or `zowe db2 call proc`, so I think it may make sense to keep 'db2' lowercase here.